src/libsync/owncloudpropagator.h

```diff
@@ -98,7 +98,14 @@ class PropagatorJob : public QObject
     virtual qint64 committedDiskSpace() const { return 0; }
 
 public slots:
-    virtual void abort() {}
+    /*
+     * Asynchronous abort requires emit of abortFinished() signal,
+     * while synchronous is expected to abort immedietaly.
+    */
+    virtual void abort(const bool asyncAbort) {
```

Comment:
parameters passed by value should not be const.   It should simply be `bool asyncAbort`


(Ideally, even, this should not be a bool, but an enum { Synchronously, Asynchronously } )

Reply:
omg, I overlooked it, of course it makes no sense, thanks! about enum, I am not sure, but definitely that would improve readability.